utility/FIOaaS/rbd/default_directory.fio

```diff
@@ -0,0 +1,15 @@
+[global]
```

Comment:
please add some doc text for this file and it's usage.

Reply:
This is not a code, its a config and self explainable for fio parameters